README.md

```diff
@@ -465,7 +465,18 @@ Tree-sitter based folding. _(Technically not a module because it's per windows a
 ```vim
 set foldmethod=expr
 set foldexpr=nvim_treesitter#foldexpr()
-set nofoldenable                     " Disable folding at startup.
+" With the above settings applied, vim will fold everything by default when
+" opening a file... setting foldlevelstart to a high value allows you to
+" start with no folds and then toggle them as desired with the za keymap.
+set foldlevelstart = 99
+```
+
+or if you prefer lua:
+
+```lua
+vim.wo.foldmethod = "expr"
+vim.wo.foldexpr = "nvim_treesitter#foldexpr()"
+vim.go.foldlevelstart = 99
```

Comment:
Not that `vim.{wo,go}` is any wrong, just maybe not intended since `set ...` can also be global, so I'd say maybe `vim.o`

Reply:
You're right... it would be preferred to set the options globally with `vim.o` so that if a user wanted to change them locally and then revert to global settings with `:set option<` they would be able to do so. I'll make that change. Thanks!

Reply:
If anything, can you also change the `foldexpr` to `vim.o.foldexpr = 'v:lua.vim.treesitter.foldexpr()'`, as `nvim_treesitter#foldexpr()` is already deprecated/planned to remove. 

See https://github.com/nvim-treesitter/nvim-treesitter/tree/main#folds

Reply:
Thanks! I was looking for a more lua way to do that but gave up on it. I'll add that as well... and a comment about setting the fold options per-filetype.